src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -2739,14 +2915,15 @@ namespace SimAirServingZones {
 		IterTot = 0;
 
 		AirLoopConvergedFlag = true;
+		BypassOAController = false; // not exactly sure of this but it seems all controllers need to be simulated -- don't bypass
```

Comment:
new flag to tell HVAC controller to either simulate or bypass if this controller is in OA system